dumper/app/src/main/java/com/google/edwmigration/dumper/application/dumper/connector/hive/HiveMetadataConnector.java

```diff
@@ -538,11 +541,7 @@ protected void run(@Nonnull Writer writer, @Nonnull ThriftClientHandle thriftCli
         ImmutableList<String> masterKeys = client.getMasterKeys();
         for (String masterKey : masterKeys) {
           monitor.count();
-          JsonGenerator jsonGenerator = jsonFactory.createGenerator(writer);
-          jsonGenerator.writeStartObject();
-          jsonGenerator.writeStringField("masterKey", masterKey);
-          jsonGenerator.writeEndObject();
-          jsonGenerator.flush();
+          writer.write(objectMapper.writeValueAsString(MasterKey.create(masterKey)));
```

Comment:
Optimization note: We could also use `objectMapper.writeValue(writer, MasterKey.create(...))` rather than allocating the String.

Reply:
`writeValue` closes the `writer`, which I don't want to do in this line.

Reply:
d'oh, thank you! I think we had that discussion before and I forgot.